src/Core/IEPlugin/IEPluginInit.cc

```diff
@@ -32,10 +32,12 @@
 #include <Core/IEPlugin/SimpleTextFileToMatrix_Plugin.h>
 #include <Core/IEPlugin/EcgsimFileToMatrix_Plugin.h>
 #include <Core/IEPlugin/PointCloudField_Plugin.h>
-#include <Core/IEPlugin/IgbFileToMatrix_Plugin.h>
+#include <Core/IEPlugin/IGBFileToMatrix_Plugin.h>
```

Comment:
@kgillette This file is not included in the PR. Looks like you commented out the line of code below that requires it. Delete this line to make the build pass.